web/src/App.tsx

```diff
@@ -79,10 +78,10 @@ export default function App(): ReactElement {
             <Sentry.ErrorBoundary fallback={ErrorComponent} showDialog>
               {isSuccess && isNewVersionAvailable && (
                 <Toast
-                  title={__('misc.newversion')}
+                  title={i18n.t('misc.newversion')}
```

Comment:
Why not use the `useTranslation` hook here?

Reply:
Good question 😅

I'll change it...

Reply:
I remember why I did it like this now...

Otherwise the i18n import becomes unused and well vite removes the whole thing 😅

It should still be the same instance though so should not impact any functionality.

Reply:
Ah okay, fair :)